<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="asklap" ignored="0" name="TestMockito" tests="4" failures="1" timestamp="27 Sep 2018 06:58:07 GMT" time="1.143" errors="0">
  <testcase name="testPerform on null(UseMockito.UseMockito.TestCalculator)" time="0.093" classname="UseMockito.UseMockito.TestCalculator"/>
  <testcase name="testPerformMul on null(UseMockito.UseMockito.TestCalculator)" time="0.011" classname="UseMockito.UseMockito.TestCalculator"/>
  <testcase name="testSum" time="0.012" classname="UseMockito.UseMockito.TestNumberSummer"/>
  <testcase name="testSum" time="0.021" classname="UseMockito.UseMockito.TestNumberSummer">
    <failure type="java.lang.AssertionError" message="Sum and expected sum are not matching expected [-3] but found [-2]">
      <![CDATA[java.lang.AssertionError: Sum and expected sum are not matching expected [-3] but found [-2]
at UseMockito.UseMockito.TestNumberSummer.testSum(TestNumberSummer.java:23)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testSum -->
</testsuite> <!-- TestMockito -->
